docs/data-sources/appliance.md

```diff
@@ -3,12 +3,12 @@
 page_title: "spectrocloud_appliance Data Source - terraform-provider-spectrocloud"
 subcategory: ""
 description: |-
-  
+  Provides details about a single appliance.
```

Comment:
This needs more information @nikchern. Explain what an appliance is.

Reply:
Sure will do @karl-cardenas-coding